### PR TITLE
Add a chapter on Coastal

### PR DESCRIPTION
**Pull request by DCEW**
Created a chapter describing the NDRS work to derive a coastal variable for cancer data reporting.

### CODE REVIEW

File: coastal.qmd [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+It is proposed that this definition is now taken up for health inequality reporting where breakdowns by coastal/inland may of interest to highlight discrepancies in provision, opportunity or outcomes. The [code used in the manuscript](https://github.com/DCEW/Coastal_BMJPH) is open source to support dissemination and collaboration.
+
+## Caveats...
+
+Although one definition would not necessarily be the best variable for describing residual variation across all disease states/outcomes, or indeed in all cancer sites, the paper posits that standardisation of approach to allow for cross-project comparisons is of greater use in public health than many different coastal definitions being applied in other scenarios.
+
+Other approaches to defining coastal geographies for epidemiological or social sciences applications have been investigated or are ongoing, e.g.
+
```
Comment (Lextuga007) [inline coastal.qmd:22]: NHS-R Way style, following Turing Way, is to avoid Latin abbreviations so e.g. needs to be for example 🙂

**Review by Lextuga007 — CHANGES_REQUESTED** [review]

File: coastal.qmd (hunk 1 above)
Comment (DCEW) [reply coastal.qmd:22]: No probs, will change and recommit :-)

Comment (DCEW) [reply coastal.qmd:22]: Done

**Review by Lextuga007 — APPROVED** [review]
Thank you! Great chapter addition to the book!

**Comment by Lextuga007** [comment]
@all-contributors please add @DCEW for content

**Pull request merged**